modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
 	compileOnly group: "org.apache.felix", name: "org.apache.felix.http.servlet-api", version: "1.1.2"
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
 	compileOnly group: "org.osgi", name: "osgi.core", version: "6.0.0"
+	compileOnly project(":apps:data-engine:data-engine-taglib")
```

Comment:
> Is this necessary? is this module depending on something from data-engine-taglib?

Si papa, since We're requiring the Calculator from the `data-engine-taglib` module on form-builder.

Reply:
Remember that this is for build time, so the `gw deploy` will be "slower", we don't need to add this dependency when we are just depending on JavaScript this is more important for a Java dependency. Adding this dependency to `npmscripts.config.js` and `.npmbundlerrc` is enough.

Reply:
Done! It's unnecessary :)